conventions/identity.md

```diff
@@ -0,0 +1,76 @@
+# Mod Identity
+The 'identity' of your mod consists of it's name, mod id, filename, and version.
+
+## Mod ID
+Mod IDs should consist of alphanumeric characters (0-9, A-Z) and be all-lowercase. For example, vanilla's "mod id" is 'minecraft'.
+
+Spacers (underscore and hyphen) are fine, but hyphens are preferred. **A mod id should never contain spaces or periods.** These can cause strange issues to arise.
+
+## Mod Name
+A mod's name can be virtually anything. Unicode, spacers, symbols, alphanumerics, all should work fine, but may trip up faulty code.
+
+To maximize compatibility, mod names should be restricted to alphanumerics (A-Z, 0-9), spaces, and basic symbols.
+Using characters outside this space is fine, but may trip up poorly-coded or legacy mods. A mod name should never contain color codes, as it can cause rendering problems.
+
+## Filenames
+A mod's filename can be anything supported by the target system, but there are some symbols and characters that can cause problems, or be outright refused, on some systems.
+A mod filename should never contain any of the characters that are illegal on most popular filesystems; these are:
+
+ - `#` (pound/hash/number sign)
+ - `<` (left angle bracket/less than)
+ - `>` (right angle bracket/greater than)
+ - `$` (dollar sign)
+ - `+` (plus sign)
+ - `%` (percent sign)
+ - `!` (exclamation point)
+ - <code>`</code> (backtick/grave)
```

Comment:
I tried a variety of ways of escaping the backtick, including your suggestion, and this is the only one that worked.